src/Components/PositionExpandableContent/PositionExpandableContent.jsx

```diff
@@ -51,10 +57,10 @@ const PositionExpandableContent = ({ sections, form }) => {
       content: (
         <div className="simple-action-modal">
           <div className="help-text">
-            <span>Are you sure you want to discard all changes made to this position?</span>
+            <span>{form.cancelText || 'Are you sure you want to discard all changes made to this position?'}</span>
           </div>
           <div className="modal-controls">
-            <button onClick={() => { setEditMode(false); swal.close(); }}>Submit</button>
+            <button onClick={() => onCancel()}>Submit</button>
```

Comment:
Just for fun - you can always shorten these callbacks without passed args to the name uninvoked 
```suggestion
            <button onClick={onCancel}>Submit</button>
```

Reply:
thanks! commited